modules/migration-network-traffic-redirection-strategies.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc
+// * migration_toolkit_for_containers/network-considerations-3-4.adoc
+
+[id="migration-network-traffic-redirection-strategies_{context}"]
+= Network traffic redirection strategies
+
+After a successful migration, you have to redirect network traffic of your stateless applications from the source cluster to the target {product-title} {product-version} cluster.
+
+All strategies for redirecting network traffic described below are based on these assumptions:
+
+* Applications are deployed on the {product-title} {product-version} cluster. The original application pods are still running on the source cluster.
+* Each application has a route that contains the source cluster hostname.
+* The route with the source cluster hostname contains a CA certificate.
+* If necessary for securing connections made through the source cluster's DNS domain, the {product-title} {product-version} router default certificate includes the source cluster wildcard Subject Alternative Name.
```

Comment:
"If necessary" is a dangler. Change to "If you require secure connections ..."

Remove "made" after connections. It is unnecessary.

Avoid anthropomorphism: "certificate includes". Is the user supposed to ensure that the certificate includes the wildcard? If this is the case, you need to say so.

I think Subject Alternative Name might need markup. It looks like a parameter. Check this.